### PR TITLE
docs: add digital ocean related posts

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Creates "Add DigitalOcean Credential" and "Add a Server to DigitalOcean" posts, with the free tier account link note added

## Issues resolved by this PR

- [x] [DVN-10913]
- [x] [DVN-10912]
- [x] [DVN-10888]

## More info

"Add DigitalOcean Credential" post preview
image

"Add a Server to DigitalOcean" post preview
image

[DVN-10913]: https://devopness.atlassian.net/browse/DVN-10913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10912]: https://devopness.atlassian.net/browse/DVN-10912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10888]: https://devopness.atlassian.net/browse/DVN-10888?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/cloud-provider-credentials/add-digitalocean-credential.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Under `Token name`, click `Enter token name` field and type a name for the new token
+    - Tip: you might want to name the user as `devopness` to make it easier to track its activities
+1. Under `Expiration`, click the `Select token expiry` field and select one of the expiration options for when this token will become expired
+1. Under `Select scopes`, ensure the `Write` option is checked
+    - It’s critical that your credential key has the permissions to create instances, otherwise Devopness will be unable to add new Servers.
+1. Click `Generate Token`
+1. Copy the value of `token`
+    - It will be the text prefixed with `dop_v1_`
+    - Be sure to record your personal access token. For security purposes, it will not be shown again.
+1. To add the copied credentials to Devopness see [Add a Cloud Provider Credential](/docs/cloud-provider-credentials/add-cloud-provider-credential/)
```
Comment (jfoliveira) [inline docs/cloud-provider-credentials/add-digitalocean-credential.md:30]: Perhaps add a final note pointing to vendor official guide on how to create a token? As this CSP offers that, that might be useful.

Something like:
```suggestion
1. To add the copied credentials to Devopness see [Add a Cloud Provider Credential](/docs/cloud-provider-credentials/add-cloud-provider-credential/)

> For further details on how to create a DigitalOcean token, please refer to [DigitalOcean documentation](https://docs.digitalocean.com/reference/api/create-personal-access-token/)

```

File: docs/servers/add-digitalocean-server.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `DigitalOcean` as cloud provider
+1. Select an DigitalOcean credential using the drop-down menu
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](/docs/cloud-provider-credentials/add-cloud-provider-credential/) guide
+1. Click `NEXT`
+1. Select an avaliable region using the drop-down menu
+1. Select an avaliable instance type using the drop-down menu
```
Comment (jfoliveira) [inline docs/servers/add-digitalocean-server.md:27]: It seems `available` has been misspelled as `avaliable` here and also in other docs. If you could fix them on this PR as well, that would be great. See list here: https://github.com/search?q=repo%3Adevopness%2Fdevopness%20avaliable&type=code

```suggestion
1. Select an available region using the drop-down menu
1. Select an available instance type using the drop-down menu
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/cloud-provider-credentials/add-digitalocean-credential.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+> If you don't have access to an DigitalOcean account, you can create an account for free following the cloud provider specific guide [DigitalOcean Free Trial Offer](https://www.digitalocean.com/try/free-trial-offer)
+
+1. Log in to [DigitalOcean Control Panel](https://cloud.digitalocean.com/)
+1. In the navigation panel on the left side, click `API`
+1. Below the title `Applications & API`, select the `Tokens` tab
+1. In the `Tokens` tab, click `Generate New Token`
+1. The `New personal access token` form will be displayed
+1. Under `Token name`, click `Enter token name` field and type a name for the new token
+    - Tip: you might want to name the user as `devopness` to make it easier to track its activities
```
[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/cloud-provider-credentials/add-digitalocean-credential.md:22]: Sorry, meant token name

File: docs/cloud-provider-credentials/add-digitalocean-credential.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+1. Under `Expiration`, click the `Select token expiry` field and select one of the expiration options for when this token will become expired
+1. Under `Select scopes`, ensure the `Write` option is checked
+    - It’s critical that your credential key has the permissions to create instances, otherwise Devopness will be unable to add new Servers.
```
[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/cloud-provider-credentials/add-digitalocean-credential.md:25]: This note was copied from the add DigitalOcean credential form in Devopness Web App.

File: docs/servers/add-digitalocean-server.md (hunk 2 above)
Comment (thlmenezes) [reply docs/servers/add-digitalocean-server.md:27]: I'll update the PR soon with the changes applied

File: docs/servers/add-digitalocean-server.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+1. Click `NEXT`
+1. Provide your server name, OS and hostname and click `NEXT`
+1. Review server information and settings. If everything is correct, click `CONFIRM`
+    > Use the `PREVIOUS` button if you need to modify any server details
+1. Wait for the `Server:provision` action to finish
+1. After the server is successfully provisioned it will be visible on `DigitalOcean` console
+1. If you want to access the server on `DigitalOcean` console:
+    - Log in to [DigitalOcean Control Panel](https://cloud.digitalocean.com/)
+    - Find the droplets section
+    - A `droplet` instance with the name provided on Devopness web interface will be listed
```
Comment (jfoliveira) [inline docs/servers/add-digitalocean-server.md:38]:
```suggestion
    - A `droplet` instance will be listed, with the name of the new `Server` you provisioned on Devopness
- ```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/servers/add-digitalocean-server.md (hunk 5 above)
Comment (thlmenezes) [reply docs/servers/add-digitalocean-server.md]: Changes applied

File: docs/servers/add-aws-server.md [hunk 6]
```diff
@@ -37,4 +37,4 @@ links:
     - On the Search box, enter “EC2” and select it from search results
     - In the upper-right corner select the region on which server was provisioned
     - Click `Instances`
-    - A `EC2` instance with the name provided on Devopness web interface will be listed
+    - A `EC2` instance will be listed, with the name of the new `Server` you provisioned on Devopness
```
Comment (jfoliveira) [inline docs/servers/add-aws-server.md:40]:
```suggestion
    - An `EC2` instance will be listed, with the name of the new `Server` you provisioned on Devopness
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**